pyproject.toml

```diff
@@ -61,7 +61,6 @@ aleph-alpha-client = {version="^2.15.0", optional = true}
 deeplake = {version = "^3.3.0", optional = true}
 pgvector = {version = "^0.1.6", optional = true}
 psycopg2-binary = {version = "^2.9.5", optional = true}
-#boto3 = {version = "^1.26.96", optional = true} # TODO: fix it, commented because the version failed with deeplake
```

Comment:
@hwchase17 FYI 

This PR

https://github.com/hwchase17/langchain/pull/2560

commented out boto3 for users. Not sure if an issue, but it left an incorrect poetry file